app_feup/lib/controller/fetchers/calendar_fetcher_html.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:http/http.dart';
+import 'package:uni/controller/networking/network_router.dart';
+import 'package:uni/controller/parsers/parser_calendar.dart';
+import 'package:uni/model/app_state.dart';
+import 'package:redux/redux.dart';
+import 'package:uni/model/entities/calendar_event.dart';
+import 'package:uni/model/entities/session.dart';
+
+/// Fetch the school calendar from HTML
+class CalendarFetcherHtml {
+  Future<List<CalendarEvent>> getCalendar(Store<AppState> store) async {
+    final String url = NetworkRouter.getBaseUrl('feup')
+      + 'web_base.gera_pagina?p_pagina=página%20estática%20genérica%20106';
```

Comment:
I noticed that after the multiple faculties PR was merged, the url for fetching the restaurants became the return of a new method, getEndpoints, which in the future will make it easier to return the different urls for users with more than one faculty: https://github.com/NIAEFEUP/project-schrodinger/blob/develop/app_feup/lib/controller/fetchers/restaurant_fetcher/restaurant_fetcher_html.dart

If there are any plans in the future to have the school calendar's display, not only from feup, but also from other faculties and possibily more than one calendar for users with more than one faculty, I think adding a getEndpoints method similar to the one above could be added here as well. Wdyt?